hack/prepare.sh

```diff
@@ -9,6 +9,9 @@ set -Eeuo pipefail
 # Use as /usr/local/sbin/dp-prepare.sh
 #
 
+OPT_MULTIPORT=false
+OPT_PF1_PROXY=false
```

Comment:
Why dont we get these values as parameters to the script ?
Smt like this

```bash
OPT_MULTIPORT=false
OPT_PF1_PROXY=false

while [[ "$#" -gt 0 ]]; do
  case $1 in
    -multiport)
      OPT_MULTIPORT=true
      ;;
    -pf1proxy)
      OPT_PF1_PROXY=true
      ;;
    *)
      echo "Invalid option: $1" >&2
      exit 1
      ;;
  esac
  shift
done
```

This can be better consumed/set in the kubernetes yaml files configuring the dpservice init container using this script.

Reply:
We do use these as a parameter of the prepare script exactly for the reason that I can then easily edit `kustomization.yaml` of the init container to switch to multiport. It also sets the `dpservice` command-line option `multiport-eswitch` inside `dp_service.conf` so there is no need to edit anything else.

Your code matches mine at the bottom of the file. This is only to set defaults, because we have the `set -u ` in place so bash complains about unset variables.

Reply:
Yeah. Friday evening review. I didnt see the code below.